Browser_IDE/stylesheet.css

```diff
@@ -472,4 +541,73 @@ li.CodeMirror-hint-active {
   position: relative;
   flex: 1 1 auto;
   padding: var(--bs-modal-padding);
-}
\ No newline at end of file
+}
+
+/* Modal UI changes */
+
+#confirmationModal .sk-header, #projectConflictModal .sk-header {
+  padding: 1.5em 1em; 
+  display: flex; 
+  align-items: center; 
+}
+
+#confirmationModal .modal-body, #projectConflictModal .modal-body {
+  border-top: 1px solid white;
+  border-bottom: 1px solid white; 
+}
+
+#confirmationModal .sk-modal-footer, #projectConflictModal .sk-modal-footer {
+  padding: 2.5em 1em; 
+  border-top: 1px solid white;
+  display: flex; 
+  align-items: center; 
+  justify-content: flex-end; 
+}
+
+
+.btn {
+  font-size: 1rem; 
+  padding: 0.4em 0.8em; 
+  border-radius: 0.25rem; 
+  margin-right: 0.5rem; 
+  margin-top: 0.5rem; 
+  transition: background-color 0.3s, box-shadow 0.3s; 
+}
+
+.btn:last-child {
+  margin-right: 0;
+}
+
+
+.btn-secondary {
+  background-color: #6c757d; 
+  color: white; 
+}
+
+.btn-secondary:hover, .btn-secondary:focus {
+  background-color: #5a6268; 
+}
+
+
+.btn-success {
+  background-color: #28a745; 
+  color: white; 
+}
+
+.btn-success:hover, .btn-success:focus {
+  background-color: #218838; 
+}
+
+/* Canvas scaling fix */
+#ExecutionEnvironment canvas {
+  max-width: 100%;
+  height: auto;
+  width: auto; 
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
This is a nice idea, but right now it doesn't really work:
![mobilenew](https://github.com/thoth-tech/SplashkitOnline/assets/42032199/f41018b9-3eb8-48a3-929c-dd6a04eed6fe)
Either remove it, or make sure it works right 😄